### PR TITLE
Charts 

### PR DESCRIPTION
**Pull request by DuarteDuarte**
need to improve draw_bars method and draw_combination method too.

### CODE REVIEW

**Comment by DuarteDuarte** [comment]
Please review my code. 

Also need help figuring out a problem with the charts, they appear small on the beginning, and after zooming in and out on the page, they get the right size...

File: app/views/stats/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+			</div>
+	    </div>
+	</div>		
+
+</div>
+
+
+
+
+
```
Comment (guterres) [inline app/views/stats/index.html.erb]: teste

File: app/views/stats/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap-theme.min.css">
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
```
Comment (guterres) [inline app/views/stats/index.html.erb]: It isn't necessary to put the css/js files here. Put it in the assets/application.css/js

Comment (DuarteDuarte) [inline app/views/stats/index.html.erb]: I did. it doesn't work that way. Help me find another solution.

File: app/helpers/stats_helper.rb [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+      ex[0] = User.find(ex[0]).email
+    end
+    user_expenses
+  end
+
+  def revenues_by_user
+    user_revenues = Revenue.all.group(:user_id).sum(:amount).to_a
+    user_revenues.each do |ex|
+      ex[0] = User.find(ex[0]).email
+    end
```
Comment (guterres) [inline app/helpers/stats_helper.rb]: try 

``` ruby
user_revenues = Revenue.joins(:user).group(:email).sum(:amount).to_a
```

What happens?

File: app/views/stats/index.html.erb (hunk 2 above)
Comment (guterres) [inline app/views/stats/index.html.erb]: Why not ?  only simply place this in assets/javascripts/application.js, for example:

``` ruby
//= require jquery.min
//= require bootstrap.min
```

About the highcharts... Click here :point_right: [:trollface:](https://github.com/PerfectlyNormal/highcharts-rails#usage)

File: app/helpers/stats_helper.rb (hunk 3 above)
Comment (DuarteDuarte) [inline app/helpers/stats_helper.rb]: It works great, going to change it

**Pull request merged**